examples/quantization/brevitas/quantize_llm.py

```diff
@@ -69,10 +78,12 @@
     is_static=args.is_static,
     weights_symmetric=True,
     activations_symmetric=args.is_static,  # ONNX export only supports unsigned for dynamic quantization
+    gpu_device_map=gpu_device_map,
+    cpu_device_map=cpu_device_map,
 )
 
 quantizer = BrevitasQuantizer.from_pretrained(
-    args.model, device_map="auto" if args.cpu_offload else "cuda:0", torch_dtype="auto"
+    args.model, device_map=args.device, # torch_dtype="auto" # fails for some reason with quantization enabled
```

Comment:
```suggestion
    args.model, device_map=args.device
```

I agree, let's ignore `torch_dtype` for now. we can fix later.